cmd/ttn-lw-cli/commands/contact_info.go

```diff
@@ -173,8 +174,36 @@ func contactInfoCommands(entity string, getID func(cmd *cobra.Command, args []st
 			return io.Write(os.Stdout, config.OutputFormat, updatedInfo)
 		},
 	}
+	validate := &cobra.Command{
+		Use:   "validate [reference] [confirmation-token]",
+		Short: "Validate contact info",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			reference, _ := cmd.Flags().GetString("reference")
+			if reference == "" {
+				return errMissingParam
+			}
+			token, _ := cmd.Flags().GetString("validation-token")
+			if token == "" {
+				return errMissingParam
+			}
+			is, err := api.Dial(ctx, config.IdentityServerGRPCAddress)
+			if err != nil {
+				return err
+			}
+			res, err := ttnpb.NewContactInfoRegistryClient(is).Validate(ctx, &ttnpb.ContactInfoValidation{
+				ID:    reference,
+				Token: token,
+			})
+			if err != nil {
+				return err
+			}
+			return io.Write(os.Stdout, config.OutputFormat, res)
+		},
+	}
+	validate.Flags().String("reference", "", "")
```

Comment:
Consider adding help text. `reference` by itself may not be clear enough

Reply:
@htdvisser would "reference of your contact info validation (received in email)" suffice? assuming email is the only possible way of receiving it. or we can just exclude the part in braces.

Reply:
Or "that you received"